src/connectors/warp-aphextwin.ts

```diff
@@ -2,6 +2,8 @@ export {};
 
 Connector.playerSelector = '.player';
 
+Connector.artistSelector = '.current-track .js-current-track-artist';
```

Comment:
Not sure we need this defined here as we're overriding the `getArtist()` function which would leverage this.

Reply:
I've added `Connector.artistSelector` for consistency with `Connector.trackSelector`, so we can quickly see where we are getting the data. `getArtist` then allows cleaning up what's in `artistSelector`